kolibri/core/assets/src/views/Navbar/NavbarLink.vue

```diff
@@ -131,10 +137,44 @@
 
   .tab-title {
     display: inline-block;
-    font-weight: bold;
     text-overflow: ellipsis;
-    text-transform: uppercase;
-    vertical-align: middle;
+  }
+
+  /deep/ .ui-icon {
+    width: 22px !important;
+    height: 22px !important;
+  }
+
+  /deep/ svg {
+    width: 20px !important;
+    height: 20px !important;
+  }
+
+  @media (max-width: 600px) {
```

Comment:
I suggest adding a comment here that the `600px` corresponds to the "small" window size for clarity for anybody coming through later on.